src/nvim/buffer_defs.h

```diff
@@ -649,7 +649,7 @@ struct file_buffer {
   bool b_p_swf;                 /* 'swapfile' */
   long b_p_smc;                 /* 'synmaxcol' */
   char_u      *b_p_syn;         /* 'syntax' */
```

Comment:
I though I'd do linting after review. Do you think I'm better doing multiple commits to split linting?